App.js

```diff
@@ -1,26 +1,151 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import { useState } from 'react'
 import styled from 'styled-components/native'
+import { Button, Share } from 'react-native'
 
 const Container = styled.View`
   flex: 1;
-  background-color: papayawhip;
   justify-content: center;
   align-items: center;
 `
 
+const Header = styled.View`
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  padding-top: 20px;
+  width: 100%;
+  background-color: #DC4730;
+`
+
 const Title = styled.Text`
   font-size: 24px;
-  color: palevioletred;
+  font-weight: 900;
+  font-style: italic;
+  color: white;
+`
+
+const SearchContainer = styled.View`
+  flex: 1;  
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 40px;
+  width: 100%;
+  background-color: #E7DFCD;  
+`
+
+const Input = styled.TextInput`
+  padding: 5px;
+  height: 40px;
+  width: 70%;
+  border-width: 1px;
+  border-radius: 8px;
+  border-color: #032A34; 
+  background-color: #E7DFCD;
+`
+
+const JokeContainer = styled.View`
+  flex: 6;
+  justify-content: center;
+  width: 100%;
+  background-color: #E7DFCD;
+`
+
+const Joke = styled.Text`
+  padding: 0 40px;
+  font-size: 20px;
+  color: #032A34;
+`
+
+const ButtonContainer = styled.View`
+  flex: 1;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 40px;
+  width: 100%;
+  background-color: #E7DFCD;
 `
 
 const App = () => {
+
+  const [joke, setJoke] = useState("")
+  const [search, setSearch] = useState()
+
+  const getJoke = () => {
+    fetch('https://icanhazdadjoke.com/', {
+      method: 'GET',
+      headers: {
+        "Accept": "application/json"
+      }
+    })
+      .then(response => response.json())
+      .then(json => setJoke(json.joke))
+  }
+
+  const searchJoke = () => {
```

Comment:
From naming point of view, `getJoke()` and `searchJoke()` functions are not very descriptive. I would rather suggest something like `getJoke()` and `getSearchedJoke()`

Reply:
makes sense, updated! 

also updated search and setSearch to searchWord and setSearchWord